test/TestApps/ToastNotificationsTestApp/main.cpp

```diff
@@ -1322,6 +1322,171 @@ bool VerifyIconPathExists_Unpackaged()
     return true;
 }
 
+bool VerifyRegisterWithNullDisplayNameFail_Unpackaged()
```

Comment:
winrt::hstring str = winrt::hstring{}; // this is like assigning nullptr to hstr but if you decode str.c_str() it would return L"". I think I would like to stick with the naming here.

Reply:
Btw there is also another unittest: VerifyRegisterWithEmptyDisplayNameFail_Unpackaged() which specifically checks for L""